bears/general/SpaceConsistencyBear.py

```diff
@@ -38,6 +55,10 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
+        if self.section.aspects:
+            indent_type = (Indentation in self.section.aspects and
```

Comment:
Should be `use_spaces` 😜 

Also this is missing the case if indent is `tab` and then `use_spaces` should became `False`